ops/docker/op-stack-go/Dockerfile.dockerignore

```diff
@@ -2,6 +2,8 @@
 # node_modules, packages (contains git submodules and node modules etc.), devnet dirs etc. all add up a lot.
 *
 
+# keep .git for multicannon
+!/.git
```

Comment:
I thought we were avoiding needing this and just pulling from a tagged docker image?

Reply:
Good catch. Leftover from a merge. Fixed.